models/customer.py

```diff
@@ -0,0 +1,44 @@
+from flask_restplus import fields
+def createCustomerSchema(api):
+
+    reservation = api.model('Reservation', {
+        'id_store' : fields.String('ID of the store.'),   
+        'date': fields.Date(required=True, description='The availability date'),
+        'time': fields.DateTime(required=True, description='The availability time')
+    })
+
```

Comment:
You shouldn't create the reservation schema here. You can import it from the models.reservation.py

What if i need to change schema for reservation. I have to modify all your changes here in customer.py as well